spec/models/generic_work_spec.rb

```diff
@@ -13,6 +13,26 @@
     end
   end
 
+  describe "suppressed?" do
+    let(:work) { described_class.new(title: ['demoname']) { |gw| gw.apply_depositor_metadata("user") } }
+    before do
+      allow(work).to receive(:state).and_return(::RDF::URI(activity_uri))
+    end
+    context "When a work's state is inactive" do
+      let(:activity_uri) { 'http://fedora.info/definitions/1/0/access/ObjState#inactive' }
+      it "returns a truthy statement" do
+        expect(work.suppressed?).to be_truthy
```

Comment:
Sure! I didn't know RSpec knew how to translate a truthy or falsey statements into #be_(blank) statements. Ill get that changed pushed up now.